src/invoices/InvoiceForm/InvoiceFormContainer.js

```diff
@@ -73,18 +81,25 @@ function InvoiceFormContainer({
 
     if (!forceSaveValues) {
       const { vendorInvoiceNo, invoiceDate, vendorId } = formValues;
+      const date = id ? moment.utc(invoiceDate).format(DATE_FORMAT) : invoiceDate;
```

Comment:
this line looks not clear the purpose. Does it mean that one var contains different data?

Reply:
Date from `DatePicker` comes in `YYYY-MM-DD` (2021-02-03), but then it converts on BE to `DateTime` format `2021-02-03T00:00:00.000+0000` and it should be converted back for correct filtering

Reply:
why format of the same field is different?

Reply:
IMO query should work with `2021-02-03T00:00:00.000+0000` as well

Reply:
I've checked. Unfortunately, it doesn't.